flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -141,6 +165,18 @@ def search_and_import_plugins_class(plugin_file_path: str, path_prefix: str, cla
     # Look for all possible plugins and import them
     # https://stackoverflow.com/questions/67631/how-can-i-import-a-module-dynamically-given-the-full-path
     # unfortunatelly very complicated, this is cpython only ??
+    """
+    Function serving to create a class that finds and imports the necessary modules.
+
+    Args:
+        plugin_file_path: Pathway to the module (str).
+        path_prefix: Pathway prefix to the module (str).
+        class_name: Name of the class (str).
+    Keyword args: 
```

Comment:
`**kwargs` go under the `Args:` section. See https://google.github.io/styleguide/pyguide.html#383-functions-and-methods.

Reply:
Can't find where this link tells you how to document the `**kwargs` besides just putting them underneath the `Args` section? I'm confused by this...

Reply:
Yeah, that's correct, the `**kwargs` needs to go under the `Args:` section, not in its own `Keyword args:` section. See https://google.github.io/styleguide/pyguide.html#doc-function-args specifically.

Reply:
Sorry...yes I understand that but what do you write next to the `**kwargs` under the `Args:` section. Like what argument description do you give?

Reply:
Np! 9 times out of 10 `**kwargs` are used to pass along arguments to another function like a callback or similar (which is the case here). For that situation I usually write `**kwargs: Further arguments passed to ....`. Looks like the keyword arguments are passed to the initialization of the class found by this function.